app/src/main/java/swati4star/createpdf/activity/RearrangeImages.java

```diff
@@ -95,7 +95,6 @@ public void onRemoveClick(int position) {
                         }
                         mImages.remove(position);
                         mRearrangeImagesAdapter.positionChanged(mImages);
```

Comment:
This is also extra change

Reply:
I didn't get you. Do you mean to remove these two lines of code?

Reply:
No the blank line which has been removed.